be/src/exprs/agg/hll_sketch.h

```diff
@@ -153,10 +176,50 @@ class HllSketchAggregateFunction final
         DCHECK(to->is_numeric());
 
         auto* column = down_cast<Int64Column*>(to);
-        column->append(this->data(state).estimate_cardinality());
+        if (UNLIKELY(this->data(state).hll_sketch == nullptr)) {
+            column->append(0L);
+        } else {
+            column->append(this->data(state).hll_sketch->estimate_cardinality());
+        }
+    }
+
+    std::string get_name() const override { return "ds_hll_count_distinct"; }
+
+private:
+    void _init_if_needed(FunctionContext* ctx, const Column** columns, AggDataPtr __restrict state) const {
```

Comment:
it seems that the state is not allocated on command.
can  this state's memory room  be accounted in mem tracker?